scrapers/StudioFOW.yml

```diff
@@ -9,29 +13,31 @@ sceneByURL:
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: //img[@class="rev-slidebg"]/@title
+      Title: //h2[contains(@class,"woodmart-title-container") and not(contains(text(),"DOWNLOADS"))]/text()
```

Comment:
alternative ( maybe not needed, i dont think any title contains the text DOWNLOADS )
```yaml
      Title: //h2[contains(@class,"woodmart-title-container") and @style]/text()
```